### DISCUSSION TITLE
How to pass glb model path as a variable to src

### DISCUSSION THREAD

**Discussion by w-o-o-h** [google/model-viewer#3395] [General]
I need to show glb model by passing its path to src as a variable "fname" but it does not evaluate/parse its value.

<script type="module" 
	src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js">
</script>

<script>
  	var fname = "/path_to_model" + ".glb";
</script>

<model-viewer
ar ar-scale="fixed"
camera-controls
autoplay 
alt="model"

src = fname

</model-viewer>

**Comment by elalish** [comment]
I don't think you can make HTML attributes work this way without a special framework. All attributes have JS equivalents, so you'll want to put all of this in your script tag: `mv.src = fname;`. You find the element with something like `const mv = document.querySelect('model-viewer');` or similar.

**Reply by w-o-o-h** [reply]
Thanks, solved!
